grpc/rust/main.rs

```diff
@@ -0,0 +1,28 @@
+extern crate protoc_rust_grpc;
```

Comment:
This was my second attempt at getting a working gRPC + Protobuf Rust compiler binary.

It works, but, ideally we would like to not write our own `main.rs`, but rather delegate it to an established Bazel Rust rules library - ie: `rules_rust` or other.